packages/story-editor/src/components/floatingMenu/elements/shared/useVideoTranscoding.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useLocalMedia } from '../../../../app';
+import useVideoTrim from '../../../videoTrim/useVideoTrim';
+import useFFmpeg from '../../../../app/media/utils/useFFmpeg';
+import useProperties from './useProperties';
+
+function useVideoTranscoding() {
+  const { resource } = useProperties(['resource']);
+  const { isTranscodingEnabled } = useFFmpeg();
+  const {
+    muteExistingVideo,
+    isResourceTrimming,
+    isNewResourceMuting,
+    canTranscodeResource,
+  } = useLocalMedia(
+    ({
+      state: { canTranscodeResource, isNewResourceMuting, isResourceTrimming },
+      actions: { muteExistingVideo },
+    }) => ({
+      canTranscodeResource,
+      isNewResourceMuting,
+      isResourceTrimming,
+      muteExistingVideo,
+    })
+  );
+
+  const { isMuted, id: resourceId = 0 } = resource;
+  const isTrimming = isResourceTrimming(resourceId);
```

Comment:
@timarney Will this line need to be changed?

Reply:
Yes, the function is being renamed in that other PR

Reply:
And as the element id.

Reply:
yes
https://github.com/GoogleForCreators/web-stories-wp/pull/10712/files#diff-d581131a68dd353c626d1714ea3b4d65115979020ed22b61c7e5d4a0a8fed0ddR96